scripts/example_runner.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024 Basalte bv
```

Comment:
Maybe location of this file should follow `scripts/west_commands/runners`?

Reply:
The example west command extension also resides in this directory 🤷

Reply:
I guess it's ok, just that I always like to provide some patterns that follow upstream.